core/arch/arm/kernel/boot.c

```diff
@@ -87,6 +89,8 @@ static unsigned long boot_arg_pageable_part __nex_bss;
 static uint32_t cntfrq;
 #endif
 
+static unsigned long transfer_list_pa;
```

Comment:
Please move this just below `boot_arg_pageable_part`. We should use the `__nex_bss` attribute too.

Reply:
Fixed.